level_maker.cpp

```diff
@@ -2287,6 +2297,9 @@ PLevelMaker LevelMaker::topLevel(RandomGen& random, optional<CreatureFactory> fo
       case SettlementType::CAVE:
         queue = dragonCaveMaker(settlement);
         break;
+      case SettlementType::HUGE_CAVERN:
+        queue = hugeCavernMaker(settlement);
```

Comment:
The size will be 35x35 as you defined, it will just use the dragonCaveMaker algorithm. But please verify, I may be wrong.